src/main/kotlin/dev/bright/vb6serializer/BinaryEncoder.kt

```diff
@@ -69,7 +69,7 @@ internal class BinaryEncoder(
     }
 
     override fun encodeString(value: String) {
-        encodeString(value, value.length)
+        encodeStringWithLength(value, value.length)
```

Comment:
Hmmm shouldn't this be `encodeStringWithoutLength`/`encodeStringVariableLength` ?

Reply:
It might be, although do we actually use it? What determines if we use `encodeString` and `encodeStringElement`?

Reply:
This actually fails some tests. We should leave it